.travis.yml

```diff
@@ -7,39 +8,41 @@ matrix:
       env: TOXENV=py27
     - python: '3.4'
       env: TOXENV=py34
+      dist: trusty
     - python: '3.5'
       env: TOXENV=py35
     - python: '3.6'
       env: TOXENV=py36
     - python: '3.7'
       env: TOXENV=py37
-      dist: xenial
-      sudo: true
     - python: 'pypy'
       env: TOXENV=pypy
+      dist: trusty
     - python: 'pypy3'
       env: TOXENV=pypy3
-    - env: TOXENV=flake8
-    - env: TOXENV=checkspelling
-
-addons:
-  apt:
-    packages:
-    - libtidy-0.99-0
-    - aspell
-    - aspell-en
+      dist: trusty
+    - env: TOXENV=flake8,checkspelling
+      addons:
+        apt:
+          packages:
+          - libtidy-0.99-0
```

Comment:
`libtidy` needs to be available for the syntax tests, not the spellchecking tests. In fact, we must have that as those tests are likely not run locally. As the [Contributing Guide][1] explains:

> Some tests require the PyTidyLib library, which depends on the HTML Tidy
library. If you do not have PyTidyLib installed, the tests which depend upon
it will be skipped. Given the difficulty in installing the HTML Tidy library
on many systems, you may choose to leave both libraries uninstalled and
depend on the Travis server to run those tests when you submit a pull
request.

[1]: https://github.com/Python-Markdown/markdown/blob/master/docs/contributing.md#development-environment

Reply:
Ok, this should be installed then globally again.